packages/jaeger-ui/src/components/DependencyGraph/DAG.test.js

```diff
@@ -13,8 +13,10 @@
 // limitations under the License.
 
 import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
```

Comment:
why do we need to use `ShallowRenderer` and not the standard `render`? Does `ShallowRenderer` actually render _more_ details?